himportscan/src/HImportScan/GHC9_4.hs

```diff
@@ -0,0 +1,102 @@
+{-#LANGUAGE CPP #-}
+
+#if __GLASGOW_HASKELL__ >= 904
```

Comment:
Note that this uses `>=` now and it might also work with later GHC versions, in case the API did not change.